### ISSUE TITLE
Explicitly state Mac comes with Git

### ISSUE THREAD

**Issue by ManoMarks** [alexellis/docker-blinkt-workshop#6]
I suggest that in the directions you explicitly state, for those not familiar with git, that if they are using a Mac they get git for free.

**Comment by alexellis** [comment]
Great feedback. We didn't have anyone confused in the workshop at Austin
but happy to add it.

**Comment by ManoMarks** [comment]
Yeah, most people will know, just if you have any more newbie folks
